docs/integrations/amazon-aws/lambda.md

```diff
@@ -184,10 +191,10 @@ Continue with the process of [enabling Provisioned Concurrency configurations](#
 
 AWS Lambda provides Provisoned Concurrency for greater control over the start up time for Lambda functions. When enabled, Provisioned Concurrency keeps functions initialized and hyper-ready to respond in double-digit milliseconds. AWS Lambda provides additional metrics for provisioned concurrency with CloudWatch.
 
-To collect these metrics in Sumo Logic, do the following:
```

Comment:
This comes under the heading "Collect Amazon Cloud Watch metrics," but we're redirecting to the same section. I'm a little confused. Do you think we should move the "To collect..." paragraph before the heading?

Reply:
Once the doc is released, it will redirect to the above sub-section (Collect Amazon CloudWatch Metrics). Moving it above the current title will place it in the same section it's pointing to.